lib/include/kframework/avm/teal/teal-fields.md

```diff
@@ -71,6 +71,10 @@ module TEAL-FIELDS
 ### `txn`/`gtxn` fields
 
 ```k
+
+  syntax TxnFieldTop ::= TxnField
+                       | TxnaField
+
```

Comment:
```suggestion
// Needed because itxn_field accepts both singular and array fields as arguments
  syntax TxnFieldTop ::= TxnField
                       | TxnaField

```